src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.tsx

```diff
@@ -1,27 +1,33 @@
 import classNames from 'classnames'
 import { useState } from 'react'
 
-import { DeviceDetails } from '../index.js'
-
 import { formatDateAndTime } from 'lib/dates.js'
 import { ArrowRightIcon } from 'lib/icons/ArrowRight.js'
+import { ClimateSettingScheduleCard } from 'lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleCard.js'
+import {
+  type CommonProps,
+  withRequiredCommonProps,
+} from 'lib/seam/components/common-props.js'
+import { NestedDeviceDetails } from 'lib/seam/components/DeviceDetails/DeviceDetails.js'
 import { useClimateSettingSchedule } from 'lib/seam/thermostats/climate-setting-schedules/use-climate-setting-schedule.js'
 import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
 import { DetailRow } from 'lib/ui/layout/DetailRow.js'
 import { DetailSection } from 'lib/ui/layout/DetailSection.js'
 import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
 import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
 
-import { ClimateSettingScheduleCard } from './ClimateSettingScheduleCard.js'
-
-export interface ClimateSettingScheduleDetailsProps {
+export interface ClimateSettingScheduleDetailsProps extends CommonProps {
   climateSettingScheduleId: string
-  onBack?: () => void
-  className?: string
 }
 
+export const NestedClimateSettingScheduleDetails = withRequiredCommonProps(
+  ClimateSettingScheduleDetails
+)
+
 export function ClimateSettingScheduleDetails({
   climateSettingScheduleId,
+  disableLockUnlock = false,
+  disableDeleteAccessCode = false,
```

Comment:
Isn't it a bit odd to need these props in this component?

Reply:
I'm seeing these pop up in the controls for the story: https://react.seam.co/?path=/docs/example-climatesettingscheduledetails--docs